k8s/ithc/common-overlay/wa/kustomization.yaml

```diff
@@ -6,6 +6,10 @@ bases:
 - ../../../namespaces/all-namespaces/nonprod-role.yaml
 patchesStrategicMerge:
 - ../../../namespaces/wa/wa-case-event-handler/ithc.yaml
+- ../../../namespaces/wa/wa-standalone-task-bpmn/ithc.yaml
+- ../../../namespaces/wa/wa-task-configuration-api/ithc.yaml
+- ../../../namespaces/wa/wa-task-management-api/ithc.yaml
+- ../../../namespaces/wa/wa-workflow-api/ithc.yaml
```

Comment:
can't see it in the PR diff but I am assuming the `ia-task-configuration` kustomization ref is there already?

Reply:
yes, I added too. Here: https://github.com/hmcts/cnp-flux-config/pull/9066/files#diff-6038c73edc3f2dd661218d6caab98dcf36817411d8fb967b1977527c6aa83dd7R9 

and here: https://github.com/hmcts/cnp-flux-config/pull/9066/files#diff-dc949e42add5d629316a1021b43a1f12eb21af83a7ad075af2cd106f6a6654f1R1